### ISSUE TITLE
synthesio.ovh.installation_template fails when using recent pyyaml

### ISSUE THREAD

**Issue by xtof-osd** [synthesio/infra-ovh-ansible-module#38]
When using recent Pyyaml (6.0), `synthesio.ovh.installation_template` task fails with 
```python
TypeError: load() missing 1 required positional argument: 'Loader'
```
This is related to the fact that yaml.load has been [deprecated in Pyyaml 5.1+](https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation).

**Comment by guillaume-gambs** [comment]
Hi, For information I try with this
content = yaml.load(stream, Loader=yaml.FullLoader)

**Comment by guillaume-gambs** [comment]
It works thanks ;)

**Comment by Skunnyk** [comment]
Fixed in #40 , thank you !
